tor/__init__.py

```diff
@@ -1,49 +1 @@
-import bugsnag
-from addict import Dict
-
 __version__ = '2.6.14'
```

Comment:
Maybe we should follow guidelines at http://keepachangelog.com here? It enforces a process for release without meaning to, making semver much easier.